packages/teams-js/src/internal/communication.ts

```diff
@@ -1020,3 +1062,20 @@ function createMessageEvent(func: string, args?: any[]): MessageRequest {
     args: args || [],
   };
 }
+
+function getMessageIdsAsLogString(
+  message:
```

Comment:
Intentional that it does not include `SerializedMessageResponse` as one of the possible types?

Reply:
It's sort of intentional in that no one is calling this function with an object of that type but it's easy enough to add it, so I did.